src/ProtectAgainstSpam.php

```diff
@@ -35,6 +35,10 @@ public function handle(Request $request, Closure $next): Response
             $nameFieldName = $this->getRandomizedNameFieldName($nameFieldName, $request->all());
         }
 
+        if ($this->continueWithMissingHoneypotFields($request, $nameFieldName)) {
```

Comment:
I think this would be much more readable if we rename the function to `shouldCheckHoneypot` and use `! $this->shouldCheckHoneypot` in this function.